File: template/async-python/function/handler.py


schema = {
    'required': [
        'name',
        'surname',
    ]
}



async def handle(body, path, query):

    return {
        'status': 200,
        'body': {
            'message': f'ciao {body.name} {body.surname}!'
        }
    }

File: template/async-node/function/handler.ts



export const schema: JsonSchema = {
    type: 'object',
    required: [
        'name'
    ]
}


export const handler: Handler = async ({ body: { name }}) => {
    // throw new Error('ho no!')
    return {
        body: {
            message: `ciaoo ${name}`
        },
        status: 200,
    }
}
